bucket/talosctl.json

```
@@ -0,0 +1,27 @@
+{
+    "version": "1.3.6",
+    "description": "CLI for out-of-band management of Kubernetes nodes created by Talos<",
+    "homepage": "https://talos.dev",
+    "license": " MPL-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siderolabs/talos/releases/download/v1.3.6/talosctl-windows-amd64.exe",
```

```suggestion
            "url": "https://github.com/siderolabs/talos/releases/download/v1.3.6/talosctl-windows-amd64.exe#/talosctl.exe",
```
You can rename the file in the URL itself